lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -27,7 +27,7 @@ def run!(args)
     end
 
     def analyze
-      @matches = @output.chomp.split("\n\n").map { |m| m.split("\n") }
+      @matches = @output.chomp.split("\n\n").map { |m| m.gsub(/\n\s+\[/, "SPLIT_ME_HERE_PLEASE[").split("SPLIT_ME_HERE_PLEASE") }
```

Comment:
I just don't like how it's done, but I now see why you're doing it this way.
```@lines.chomp.split("\n\n").map { |m| m.split(/\n\s+\[/) }``` might be sufficient? You'll be missing the opening bracket, but it's not really needed if the ```REEK_REGEX``` is modified accordingly?